tests/testthat/_snaps/query-semi-join.md

```diff
@@ -24,7 +24,7 @@
       SELECT * FROM `df` AS `LHS`
       WHERE EXISTS (
         SELECT 1 FROM `df` AS `RHS`
-        WHERE (`LHS`.`x` = `RHS`.`x` AND `LHS`.`y` = `RHS`.`y`)
+        WHERE (`LHS`.`x` = `RHS`.`x`) AND (`LHS`.`y` = `RHS`.`y`)
```

Comment:
The new parantheses are because `sql_query_semi_join.DBIConnection()` now uses `sql_clause_where()` instead of directly using `WHERE` in its `build_sql()` clause.

Reply:
LGTM